### DISCUSSION TITLE
$redirect-if-allowed

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2995] [Ideas] [answered]
some lists use exclusions to fix issues
we could be efficiently improve privacy by adding `$redirect-if-allowed` -- similar to `redirect-rule`, but only gets applied if there already is an applied green-lighting filter
example usage: `google-ima.js` -- example filter list: https://github.com/tcptomato/ROad-Block/blob/master/road-block-filters-light.txt

**Comment by ghost** [comment]
Exceptions are used when redirection breaks webpage.

**Reply by ghost** [reply]
not always

**Comment by gorhill** [comment, marked as answer]
Just use `important,redirect=...` if you disagree with those exceptions.

**Reply by ghost** [reply]
I mean yes, but that requires tedious, manual work (for every allowing filter), but anyway, thanks for taking the time to respond
